Tests/SwiftLintFramework/IntegrationTests.swift

```diff
@@ -51,6 +51,7 @@ extension String {
             let rawPointer = $0._rawValue
             let byteSize = lengthOfBytesUsingEncoding(NSUTF8StringEncoding)._builtinWordValue
             let isASCII = true._getBuiltinLogicValue()
+            // swiftlint:disable:next variable_name
```

Comment:
Test failed on Travis-CI https://travis-ci.org/realm/SwiftLint/builds/123487152 both of Xcode and SPM.
> SwiftLintFrameworkTests.IntegrationTests
  testSwiftLintLints, failed - Variable name should only contain alphanumeric characters: '_builtinStringLiteral'
  /Users/travis/build/realm/SwiftLint/Tests/SwiftLintFramework/IntegrationTests.swift:54
```
            let isASCII = true._getBuiltinLogicValue()
            let staticString = StaticString(_builtinStringLiteral: rawPointer, byteSize: byteSize,
                isASCII: isASCII)
```

Reverted again.